test/functional/feature_filelock.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+# Copyright (c) 2018 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Verify that it could not start two bitcoind in the same datadir"""
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import assert_equal
+
+import os
+
+class FilelockTest(BitcoinTestFramework):
+    def set_test_params(self):
+        self.setup_clean_chain = True
+        self.num_nodes = 2
+
+    def get_node_output(self, *, index, ret_code_expected):
```

Comment:
Move to node class as `get_output`? If that's controversial then extract to a module function for now? Then below:

```py
_, output = get_node_output(node=self.nodes[1], ret_code_expected=1)
```

Reply:
Done